### PR TITLE
Did up landing page

### PR DESCRIPTION
**Pull request by chewyxx**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import { Route, Routes } from "react-router-dom";
-import { Login, Signup } from "./Pages";
-import Home from "./Pages/Home";
+import { Login, Signup, Home } from "./Pages";
+import Landing from "./Pages/Landing";
```
Comment (inezkok) [inline client/src/App.js:3]: I think can just add Landing into the import statement above

File: client/src/Pages/Landing.jsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                    component="a"
+                    href="/"
+                    sx={{
+                    mr: 2,
+                    display: { xs: 'none', md: 'flex' },
+                    flexGrow: 1,
+                    fontFamily: 'roboto',
+                    fontWeight: 800,
+                    letterSpacing: '.2rem',
+                    color: 'black',
```
Comment (inezkok) [inline client/src/Pages/Landing.jsx:39]: i think the font color can be white?

File: client/src/Pages/Landing.jsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                        onClick={handleRegister}
+                    >
+                        Register
+                    </Button>
+                </Box>
+                </Toolbar>
+            </Container>
+            </AppBar>
+
+            <h1>Welcome to BonVoyage</h1>
```
Comment (inezkok) [inline client/src/Pages/Landing.jsx:65]: i think same for the welcome message, maybe can make it white so it's brighter?

**Review by inezkok — APPROVED** [review]
overall, i think it looks good! i think maybe might need to make the form for login and signup a little wider (looks a lil skinny)

File: client/src/App.js (hunk 1 above)
Comment (chewyxx) [reply client/src/App.js:3]: I TRIED INITIALLY but it doesnt work idk whyy :(

**Review by inezkok — APPROVED** [review]
lgtm

**Pull request merged**